src/components/Components/ComponentListView.tsx

```diff
@@ -73,7 +73,7 @@ const ComponentListView: React.FC<React.PropsWithChildren<ComponentListViewProps
   const componentPACStates = usePACStatesForComponents(components);
 
   const componentsWithLatestBuild = React.useMemo(() => {
-    if (!componentsLoaded || componentsError || !pipelineRunsLoaded || pipelineRunsError) {
```

Comment:
why the removal of pipelineRunsLoaded. Are we sure they are loaded?

Reply:
We dont want to wait for pipeline runs to load.